src/main/java/teammates/ui/template/FeedbackSessionsForm.java

```diff
@@ -0,0 +1,291 @@
+package teammates.ui.template;
+
+import java.util.List;
+
+/**
+ * Data model for the form for creating a new feedback session.
+ *
+ */
+public class FeedbackSessionsForm {
```

Comment:
This class has quite a number of attributes.
Would it be a good idea to split it into multiple classes for each subsection, maybe implemented as inner classes?
If the attributes are not too strongly coupled, it may help in organising the initialisation of the class from other classes as well (creating the subsections first before passing them in here).